apps/demo/src/h5wasm/DropZone.tsx

```diff
@@ -47,23 +49,24 @@ function DropZone(props: Props) {
     >
       <input {...getInputProps()} accept={EXT.join(',')} />
       <div className={styles.inner}>
-        {isDragActive ? (
-          <p className={styles.dropIt}>Drop it!</p>
-        ) : (
-          <>
-            <p className={styles.content}>
-              Drop an HDF5 file here{' '}
-              <button className={styles.btn} type="button" onClick={open}>
-                or use a file picker instead
-              </button>
+        {isDragActive && <p className={styles.dropIt}>Drop it!</p>}
+        <div hidden={isDragActive}>
```

Comment:
I hide the form instead of unmounting/remounting it to avoid re-triggering requests in `UrlForm`.